sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -1569,15 +1569,17 @@ def test_dir_and_file_properties_owner_group_permissions(self, **kwargs):
         file_client1.create_file()
 
         directory_properties = directory_client.get_directory_properties()
-        file_properties = file_client1.get_file_properties()
+        file_properties = file_client1.get_file_properties(upn=True)
 
         # Assert
         assert directory_properties['owner'] is not None
         assert directory_properties['group'] is not None
         assert directory_properties['permissions'] is not None
+        assert directory_properties['acl'] is not None
```

Comment:
Did you mean to add `upn=True` to `get_directory_properties`? Or did we want one with and one without?

Likely when we actually test this we'll need a new test anyway.

Reply:
Test cases are not written yet -- will round back to this
As for `get_directory_properties`, I just added it there. I am not exactly sure if it should apply (I feel like it should) but the spec isn't exactly 100% clear. If anything this is an easy revert after archboard feedback